controller/linodeobjectstoragebucket_controller_test.go

```diff
@@ -24,6 +24,7 @@ import (
 
 	"github.com/linode/linodego"
 	"go.uber.org/mock/gomock"
+	"gopkg.in/yaml.v2"
```

Comment:
Can you use `sigs.k8s.io/yaml` instead? It has better support for parsing nested fields.